app/example-page/page.tsx

```diff
@@ -0,0 +1,11 @@
+import ExampleComponent from '../../components/ExampleComponent/ExampleComponent';
```

Comment:
Also it would be nice if everyone used absolute paths that Next.js gives us, just to make them more readable, so in this case `@/components/ExampleComponent/ExampleComponent`